README.md

```diff
@@ -123,7 +123,13 @@ bub.age
 NoMethodError: private method `age' called for #<Cat:0x000001020bfce0 @name="Lil Bub", @age=2>
 ```
 
-Let's make a method in the `Cat` class that implicitly receives age, through the only way it can: `self`. A cat can have a birthday, where it ages 1 year when the birthday happens. 
+### Why Have A Private `attr_accessor`?
+
+Previously we were using getters/setters so the outside world could access our instance variables. But now we are using them so that we can locally access them. Why is that valuable? A lot of software bugs come from the incorrect modification of state, aka variables. When we are debugging our program, because so many bugs come from modification of variables, if we can centralize all modification into one place we help with debugging. That's the beauty of enforcing getters/setters. Want to see all the places an instance variable is modified? With a private `attr_accessor` you can just override it and put a `binding.pry`.
+
+Another nice feature of using setters/getters is future proofing. Maybe in the future you want to add validations whenever someone sets one of your instance variables? No problem! Just override the setter method. No need to find all the places that you modify that instance variable and change it.
+
+Let's make a method in the `Cat` class that implicitly receives age, through the only way it can: `self`. A cat can have a birthday, where it ages 1 year when the birthday happens.
```

Comment:
remove "implicitly receives age" it's not implicit at all. It's explicitly defined in the method defiition